### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by jdidion** [patmagee/WDL4j#1]
You're probably already aware that we're using your grammar in wdlTools (https://github.com/dnanexus-rnd/wdlTools). Among other tools, I am working on a WDL code formatter. For this to work correctly, I need to be able to preserve the comments that are in the source file. Does it make sense for you to add comment support here first, or should I just figure out how to do it myself in wdlTools?

Thanks!

**Comment by patmagee** [comment]
hey @jdidion did you end up figuring out the comments on your own?

**Comment by jdidion** [comment]
Yep - sorry, should have closed this. I borrowed the idea from Mike Lin in miniwdl. The code is here if you want to use it: https://github.com/dnanexus-rnd/wdlTools/blob/master/src/main/scala/wdlTools/syntax/Antlr4Util.scala
